### PR TITLE
Compliance section

### PR DESCRIPTION
**Pull request by lbadenhop**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Enter a brief description of your changes to communicate to the reviewers what you changed and why.

## Deploy preview link

Please add the deploy preview link to the **specific page** that you've changed.

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | bc948f6 |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-303.amplify-central.opendocs-builder.pcloud.axway.int |

File: content/en/docs/manage_compliance/_index.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+title: Manage your compliance validation
+linkTitle: Manage your compliance validation
+weight: 475
+date: 2023-05-04
+hide_readingtime: true
+---
+
+With the growth of API usage, there is a need for make sure the APIs are technically correct and meet the API standards of your organization. 
+The Amplify API Management Platmform allows you to validate how well your APIs meet the API guidelines of your organization.   This is known as API linting and has been integrated into the Amplify API Management.  Compliance Validation is an entitlement you can purchase API Management.
```
Comment (feons) [inline content/en/docs/manage_compliance/_index.md:10]: 'Compliance Validation is an entitlement you can purchase API Management.' ??

File: content/en/docs/manage_compliance/_index.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+date: 2023-05-04
+hide_readingtime: true
+---
+
+With the growth of API usage, there is a need for make sure the APIs are technically correct and meet the API standards of your organization. 
+The Amplify API Management Platmform allows you to validate how well your APIs meet the API guidelines of your organization.   This is known as API linting and has been integrated into the Amplify API Management.  Compliance Validation is an entitlement you can purchase API Management.  
+
+## What is compliance validation
+
+Compliance validation is the verification of your APIs in terms of how thy meeet the guidelines defined in a rule set.  Both an API Design and Security rule set can be applied to all the APIs in a selected environment.  Amplify has integrated an open-source tool (Spectral) which supports the linting of Open API version 2, Open API version 3, and Async API specification files.
```
Comment (feons) [inline content/en/docs/manage_compliance/_index.md:14]: thy?

File: content/en/docs/manage_compliance/_index.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+For example, an API can be checked for compliance in terms of the API methods supported or the presence of a security policy.  The gradiing results of the compliance are measured in the number of Errors, Warnings, Info, or Hints found.
```
Comment (feons) [inline content/en/docs/manage_compliance/_index.md:15]: typo: gradiing?

File: content/en/docs/manage_compliance/_index.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+## How to perform compliance validation
+
+API Compliance Validation can be enabled/disabled per Environment.   Navigate to the Topology screen then create/edit an environment.   The environment wizard with have a **Compliance Profile** step, where you can enable either of Design Rules or Security Rules for compliance.   A default Design rule set (default-design-reulate) and a default Security rules set (default-security-ruleset) is available to be used.  Changes take effect as soon as the environment settings are saved.
```
Comment (jquick-axway) [inline content/en/docs/manage_compliance/_index.md:19]: Sentence "The environment wizard `with` have a Compliance Profile..." should be "The environment wizard `will` have a Compliance Profile...". Notice the change from "with" to "will".

**Review by jquick-axway — CHANGES_REQUESTED** [review]

**Comment by lbadenhop** [comment]
@robertengaxway I created the table and fixed a few typos. I review/edit once you are done.

File: content/en/docs/manage_compliance/_index.md (hunk 1 above)
Comment (robertengaxway) [reply content/en/docs/manage_compliance/_index.md]: Feon, I changed the wording,  Is that better?

File: content/en/docs/manage_compliance/_index.md (hunk 2 above)
Comment (robertengaxway) [reply content/en/docs/manage_compliance/_index.md]: fixed

File: content/en/docs/manage_compliance/_index.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (robertengaxway) [reply content/en/docs/manage_compliance/_index.md]: fixed

[the remark this answers is not in the document]
Comment (robertengaxway) [reply content/en/docs/manage_compliance/_index.md]: fixed

File: content/en/docs/manage_compliance/_index.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+**D** = no errors found in the API.   Count of more than 5 warnings found in the API.   The count of info and hints found the API are ignored.
+
+**E** = A count of 1-5 errors found in the API.  The count of warnings, info and hints found the API are ignored.
+
+**F** = A count of more than 5 errors found in the API.  The count of warnings, info and hints found the API are ignored.
+
+## Grading Results
+
+In the Services Registry of the WebUI, there are columns with the Compliance Grading of each API.   Hover over the grading letter to view the counts.
```
[the remark this answers is not in the document]
Comment (robertengaxway) [reply content/en/docs/manage_compliance/_index.md]: fixed

File: content/en/docs/manage_compliance/_index.md (hunk 3 above)
Comment (robertengaxway) [reply content/en/docs/manage_compliance/_index.md]: fixed

File: content/en/docs/manage_compliance/_index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (robertengaxway) [reply content/en/docs/manage_compliance/_index.md]: fixed

File: content/en/docs/manage_compliance/_index.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
+---
+title: Manage your compliance validation
+linkTitle: Manage your compliance validation
+weight: 475
+date: 2023-05-04
+hide_readingtime: true
+---
+
+With the growth of API usage, there is a need for make sure the APIs are technically correct and meet the API standards of your organization.
```
[the remark this answers is not in the document]
Comment (robertengaxway) [reply content/en/docs/manage_compliance/_index.md]: fixed

File: content/en/docs/manage_compliance/_index.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+
+## What is compliance validation
+
+Compliance validation is the verification of your APIs in terms of how they meet the guidelines defined in a rule set.  Both an API Design and Security rule set can be applied to all the APIs in a selected environment.  Amplify has integrated an open-source tool (Spectral) which supports the linting of Open API version 2, Open API version 3, and Async API specification files.
+For example, an API can be checked for compliance in terms of the API methods supported or the presence of a security policy.  The grading results of the compliance are measured in the number of Errors, Warnings, Info, or Hints found.
+
+## How to perform compliance validation
+
+API Compliance Validation can be enabled/disabled per Environment.   Navigate to the Topology screen then create/edit an environment.   The environment wizard has a **Compliance Profile** step, where you can enable either of Design Rules or Security Rules for compliance.   A default Design rule set (default-design-ruleset) and a default Security rules set (default-security-ruleset) is available to be used.  Changes take affect as soon as the environment settings are saved.
```
Comment (feons) [inline content/en/docs/manage_compliance/_index.md:19]: Notice that we use `environment` in certain places and `Environment` in others, wondering if that's intentional?

**Review by cbordier — APPROVED** [review]
All good to me

**Review by jquick-axway — APPROVED** [review]

**Review by robertengaxway — APPROVED** [review]
approved

**Pull request merged**